src/ucp/proto/proto_multi.inl

```diff
@@ -66,6 +66,11 @@ ucp_proto_multi_max_payload(ucp_request_t *req,
     size_t max_frag = lpriv->max_frag - hdr_size;
     size_t max_payload;
 
+    ucs_assertv(lpriv->max_frag > hdr_size,
```

Comment:
Is this done in https://github.com/openucx/ucx/pull/9211?

Reply:
Yes, it is.
This pull request mark as `WIP-DNM`. It will be ready to review after I'll remove the label.
This cherry-pick was added just in an attempt to catch the unstable reproducible issue.